src/vstest.console/CommandLine/Executor.cs

```diff
@@ -410,12 +417,27 @@ private void PrintSplashScreen(bool isDiag)
             }
         }
 
-        string commandLineBanner = string.Format(CultureInfo.CurrentUICulture, CommandLineResources.MicrosoftCommandLineTitle, assemblyVersion);
+        string assemblyVersionAndArchitecture = $"{assemblyVersion} ({_processHelper.GetCurrentProcessArchitecture().ToString().ToLowerInvariant()})";
+        string commandLineBanner = string.Format(CultureInfo.CurrentUICulture, CommandLineResources.MicrosoftCommandLineTitle, assemblyVersionAndArchitecture);
         Output.WriteLine(commandLineBanner, OutputLevel.Information);
         Output.WriteLine(CommandLineResources.CopyrightCommandLineTitle, OutputLevel.Information);
+        PrintWarningIfRunningEmulatedOnArm64();
         Output.WriteLine(string.Empty, OutputLevel.Information);
     }
 
+    /// <summary>
+    /// Display a warning if we're running the runner on ARM64 but with a different current process architecture.
+    /// </summary>
+    private void PrintWarningIfRunningEmulatedOnArm64()
+    {
+        if (Path.GetFileName(_processHelper.GetCurrentProcessFileName()) == NonARMRunnerName &&
```

Comment:
Please use NonARM64RunnerName, ARM and ARM64 are different and this is unnecessarily confusing.

Reply:
The idea was be clear that it's `vstest.console.exe` (x64/x86) if we ship arm I think we'll go for `vstest.console.arm.exe`, but I'm ok with `NonARM64RunnerName`